backend/app/models.py

```diff
@@ -15,6 +15,21 @@
 from django.conf import settings
 
 
+class MapSquare(models.Model):
+    """
+    This model contains data about a specific Map Square, which includes its name,
+    a list of foreign keys to all the Photos that are in this Map Square, and
+    the boundary of this Map Square.
+    """
+    name = models.CharField(max_length=252)
+    number = models.IntegerField(null=True)
+    boundaries = models.CharField(max_length=252)
+    coordinates = models.CharField(max_length=252)
+
+    class Meta:
+        app_label = 'user_app'
```

Comment:
I am not sure what's going on here! Is there a reason you are adding `app_label = 'user_app'`? It should be in app

Reply:
Before I added that line, there was an error that said "Model class app.models.MapSquare doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS." This line seemed to resolve the error.